tests/unit/components/playback/controls/SettingsForm.spec.js

```diff
@@ -326,8 +384,62 @@ describe("SettingsForm.vue", () => {
     /* https://lmiller1990.github.io/vue-testing-handbook/testing-emitted-events.html#write-a-component-and-test   */
     wrapper.vm.onSaveCustomerId(add_customer);
 
-    expect(wrapper.vm.customerid).toEqual("Customer account -3");
-    expect(wrapper.vm.userid).toEqual("");
+    expect(wrapper.vm.entrykey_customer).toEqual("Customer account -3");
+    expect(wrapper.vm.entrykey_user).toEqual("");
+  });
+  test("When mounting SettingsForm on init, the Settings now recevies an event 'cancel-id' with no data of the added customer, validate its handled and dialog is closed'", async () => {
```

Comment:
It looks like this file is still being skipped by our ESLint.  When we go in and refactor a section of our old code, we should remove it from the "ignore" list in `.pre-commit-config.yaml` so that we can slowly bring all of our codebase up to the lint standards. If we do this a little bit at a time, then soon all of our old files will be able to be removed from the "ignore" list and our whole codebase will be easier to read for new members of the team.

Reply:
Sure, will un-comment the .pre-commit-config.yaml and verify the EsLint errors.